lib/chef/provisioning/chef_provider_action_handler.rb

```diff
@@ -42,16 +42,23 @@ def report_progress(description)
       provider.converge_by description do
         # We already did the action, but we trust whoever told us that they did it.
       end
+      puts "\n"
     end
 
     def performed_action(description)
       provider.converge_by description do
         # We already did the action, but we trust whoever told us that they did it.
       end
+      puts "\n"
```

Comment:
I think we should revert the info log part of the fix and fix it in Chef.  It should be possible.  Maybe :)